qadence_protocols/measurements/protocols.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import importlib
+from dataclasses import dataclass
+from functools import partial
+
+from qadence import QuantumModel
+from torch import Tensor
+
+from qadence_protocols.protocols import Protocol
+
+PROTOCOL_TO_MODULE = {
+    "tomography": "qadence_protocols.measurements.tomography",
+}
+
+
+@dataclass
+class Measurements(Protocol):
+    TOMOGRAPHY = "tomography"
+
+    def __init__(self, protocol: str, options: dict = dict()) -> None:
+        super().__init__(protocol, options)
+
+    def __call__(
+        self,
+        model: QuantumModel,
+        param_values: dict[str, Tensor] = dict(),
+    ) -> Tensor:
+        """Compute expectation values via measurements.
+
+        Args:
+            model (QuantumModel): Model to evaluate.
+            param_values (dict[str, Tensor], optional): Parameter values. Defaults to dict().
+
+        Returns:
+            Tensor: Expectation values.
+        """
+        try:
+            module = importlib.import_module(PROTOCOL_TO_MODULE[self.protocol])
+        except KeyError:
```

Comment:
Will it capture the possible error ? In Qadence, we also capture `ModuleNotFound` and `ImportError` see [here](https://github.com/pasqal-io/qadence/blob/f4c9957f72dc43cacea1267ed8e4df3c550f1c56/qadence/extensions.py#L24) for instance. This should also be tested.

Reply:
Added an extra except for this now